contrib/redis/rueidis/rueidis.go

```diff
@@ -0,0 +1,281 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2016 Datadog, Inc.
+
+// Package rueidis provides tracing functions for tracing the redis/rueidis package (https://github.com/redis/rueidis).
+package rueidis
+
+import (
+	"context"
+	"net"
+	"strconv"
+	"strings"
+	"time"
+
+	"github.com/redis/rueidis"
+	"gopkg.in/DataDog/dd-trace-go.v1/ddtrace/ext"
+	"gopkg.in/DataDog/dd-trace-go.v1/ddtrace/tracer"
+	"gopkg.in/DataDog/dd-trace-go.v1/internal/telemetry"
+)
+
+const (
+	componentName      = "redis/rueidis"
+	defaultServiceName = "redis.client"
+)
+
+func init() {
+	telemetry.LoadIntegration(componentName)
+	tracer.MarkIntegrationImported("github.com/redis/rueidis")
```

Comment:
Remember to also update the `ddtrace/tracer/option.go` file, specifically the `contribIntegrations` variable, with new integrations! And also make sure the relevant tests in `option_test.go` are updated and passing.